### PR TITLE
Add detail of GUI terminals and Xterm/rxvt

### PR DESCRIPTION
**Pull request by N0NB**
Enhance the manual to explain basic difference between GUI and
xterm/rxvt terminal emulators.

### CODE REVIEW

File: doc/Manual.md [hunk 1]
```diff
@@ -124,17 +124,40 @@ opening the main screen.
...
+>
+> In contrast, the older terminal emulators
+> [**XTerm**](https://invisible-island.net/xterm/) and
+> [**rxvt-unicode**](http://software.schmorp.de/pkg/rxvt-unicode.html) are
+> configured through the classic X Window System resource databse.  The `doc`
+> directory contains a sample `Xresources` file to change the colors to those
+> resembling the VGA pallete and the text font and its size from the defaults of
+> each emulator.  Comments in this file explain how to use it in X and Wayland
+> sessions.  These emulators are useful since they do not reserve many of the
+> key combinations available in Tlf such as F11 or Ctrl-PgUp/Ctrl-PgDown as most
```
Comment (zcsahok) [inline doc/Manual.md:153]: just a minor one: Tlf -> TLF

File: doc/Manual.md [hunk 2]
```diff
@@ -124,17 +124,40 @@ opening the main screen.
...
+> (GUI).  Through their GUI configuration dialogs such things as the text font,
+> text size, and foreground and background color pallete may be modified.  The
+> one setting that's definitely worth playing with is the text size, which in
+> the Ubuntu terminal can be changed from the "hamburger" menu in the top right
+> of the window.
+>
+> In contrast, the older terminal emulators
+> [**XTerm**](https://invisible-island.net/xterm/) and
+> [**rxvt-unicode**](http://software.schmorp.de/pkg/rxvt-unicode.html) are
+> configured through the classic X Window System resource databse.  The `doc`
```
Comment (dl1jbe) [inline doc/Manual.md:148]: databse -> database

**Review by dl1jbe — APPROVED** [review]
Beside the typos it looks good to me.

**Comment by N0NB** [comment]
Typos fixed, I hope!

**Review by zcsahok — APPROVED** [review]

**Comment by zcsahok** [comment]
Thanks!

**Pull request merged**